### ISSUE TITLE
Crashes after editing layout

### ISSUE THREAD

**Issue by rpform** [xLightsSequencer/xLights#2764]
This has happened multiple times after editing a layout.  I have imported some purchased sequences and after mapping everything I run through the sequence to see what doesn't look right.  Some have required me to edit my groups, usually by removing somethings.  I usually Clone the group and since I already have the sequence mapped I just rename the old on by adding something like "whole display - singing tree"  I then rename the clone to the old name.   Everything is fine at that point.  When I switch back to the Sequence and either tell it to render or play is when xlight crashes.  I then relaunch xlights and everything is fine and my changes have been made.  This is what was done when the crash report attached was created.

It also has crashed when making other changes in the layout, like creating new submodels and the like and as soon as I get back to my sequence it crashes.

Thisis on 2021.25 64 bit 
Windows 10

[xLights_dbgrpt-37600-20210822T161023.zip](https://github.com/smeighan/xLights/files/7028163/xLights_dbgrpt-37600-20210822T161023.zip)

**Comment by keithsw1111** [comment]
I am really interested in finding the cause of this ... are you able to make a video of you crashing it this way. I want to be able to see exactly what you are doing.

**Comment by rpform** [comment]
This is very odd.  Its happen to me just about everytime I edited my layout for the past week, to a point that I just planned on it and to play it safe always saved the sequence right before I went to edit just to be safe.  Today, made two changes and it worked.  I'll keep recording everytime I go into my layout until it happens.  surely it will pop up again within a day or two.

**Comment by tmitchellgcio** [comment]
Yep I can confirm this has happened to me a few times. I had to change the input on my column matrices to the bottom and not the top and now I can barely open Layout or sequencer. This has happened before when I added a group. Very strange. It remedies itself, but not for a long time. I wish I knew what it was doing and why. I'm on windows btw. Doesn't crash, mostly freezes forever

![Capture](https://user-images.githubusercontent.com/84803567/136093070-9e4279e3-15c1-4cfd-8d21-60f31a041475.PNG)
